jib-gradle-plugin/README.md

```diff
@@ -271,14 +271,16 @@ Property | Type | Default | Description
 --- | --- | --- | ---
 `jib.httpTimeout` | `int` | `20000` | HTTP connection/read timeout for registry interactions, in milliseconds. Use a value of `0` for an infinite timeout.
 `jib.useOnlyProjectCache` | `boolean` | `false` | If set to true, Jib does not share a cache between different Maven projects.
-`jib.baseImageCache` | `File` | `[user cache home]/google-cloud-tools-java/jib` | Sets the directory to use for caching base image layers. This cache can (and should) be shared between multiple images.
+`jib.baseImageCache` | `File` | `[user cache home]/google-cloud-tools-java/jib`\*\*\* | Sets the directory to use for caching base image layers. This cache can (and should) be shared between multiple images.
```

Comment:
Hmm... it is not clear what `[user cache home]` is. I guess, for example, it is either `$HOME/.cache/google-cloud-tools-java/jib` or `$XDG_CACHE_HOME/jib`? I actually forgot the exact path.

Reply:
Here `[user cache home]` means either `$XDG_CACHE_HOME` or `$HOME/.cache/`. Writing out of the full directory for everything gets verbose pretty quickly so I kept the shorthand. But if you think it's necessary to be more explicit I can change it.

Reply:
I agree. How about `[platform-dependent cache root]` instead of `[user cache home]`?

Reply:
This is specifically linux; the footnote goes into other platforms. I'll change it to be clear.